e2e/pages/CreatePage.js

```diff
@@ -3,19 +3,18 @@ import { By, until } from 'selenium-webdriver';
 module.exports = (url) => (driver) => ({
     elements: {
         appLoader: By.css('.app-loader'),
-        titleInput: By.css(`.create-page input[name='title']`),
-        teaserInput: By.css(`.create-page textarea[name='teaser']`),
+        input: (type, name) => By.css(`.create-page ${type}[name='${name}']`),
```

Comment:
Inverse the order of the arguments, and give `type` a default value of `text`. That way, for the most common cases, you'll just have to pass a `name` and no `type`.